projects/ccd-case-ui-toolkit/src/lib/shared/components/dialogs/document-dialog/document-dialog.component.html

```diff
@@ -1,16 +1,15 @@
-
 <div>
   <div class="dialog-header">
     <h2 (click)="cancel()" class="heading-h2 x">X</h2>
   </div>
   <div>
-    <h2 class="heading-h2 dialog-title">Are you sure you want to replace the existing file?</h2>
+    <h2 class="heading-h2 dialog-title">{{'Are you sure you want to replace the existing file?' | rpxTranslate}}</h2>
   </div>
   <div class="dialog-info">
-    <span class="text-info">You are about to delete the original file uploaded. Are you sure you want to proceed?</span>
+    <span class="text-info">{{'You are about to delete the original file uploaded.Are you sure you want to proceed?' | rpxTranslate}}</span>
```

Comment:
@DavidJayakumar - the space between the full stop and the next sentence should have been preserved

Reply:
done